os_windows.go

```diff
@@ -136,6 +136,9 @@ func setDefaults() {
 
 	gOpts.cmds["doc"] = &execExpr{"!", "%lf% -doc | %PAGER%"}
 	gOpts.keys["<f-1>"] = &callExpr{"doc", nil, 1}
+
+	gOpts.cmds["maps"] = &execExpr{"!", `%lf% -remote "recv %id% maps" | %PAGER%`}
```

Comment:
I'm worried that using `!` with a pager here might be a little confusing for people using `less` or something like that as their `%PAGER%`.

Perhaps we should remove the pager on Windows entirely and let people configure it if they want? Hopefully, most terminals will have scrollback.

We could also hard-code `more` as the pager, but I think I prefer just not having anything.

Reply:
OTOH, this problem would also affect `:doc`, and I don't remember many complaints. We can keep it as is for now and see if complaints appear.